msbuild/Xamarin.iOS.Tasks.Core/Tasks/CreateDebugConfigurationTaskBase.cs

```diff
@@ -44,29 +42,32 @@ public override bool Execute ()
 			Log.LogTaskProperty ("SdkIsSimulator", SdkIsSimulator);
 
 			var ips = DebugIPAddresses?.Split (new char [] { ';' }, StringSplitOptions.RemoveEmptyEntries);
+			var path = Path.Combine (AppBundleDir, "MonoTouchDebugConfiguration.txt");
+			var added = new HashSet<string> ();
+			var builder = new StringBuilder ();
 
-			try {
-				using (var stream = new FileStream (path, FileMode.Create, FileAccess.Write)) {
-					using (var writer = new StreamWriter (stream)) {
-						var added = new HashSet<string> ();
-						if (ips != null) {
-							foreach (var ip in ips) {
-								string str = ip.ToString ();
-
-								if (added.Contains (str))
-									continue;
-
-								writer.WriteLine ("IP: {0}", str);
-								added.Add (str);
-							}
-						}
-
-						if (!DebugOverWiFi && !SdkIsSimulator)
-							writer.WriteLine ("USB Debugging: 1");
-
-						writer.WriteLine ("Port: {0}", DebuggerPort);
-					}
+			if (ips != null) {
+				foreach (var ip in ips) {
+					if (added.Contains (ip))
+						continue;
+
+					builder.AppendFormat ("IP: {0}", ip);
```

Comment:
minor but `AppendFormat` allocates memory to create a new `string` first. The same can be avoided by

```
builder.Append ("IP: ");
builder.AppendLine (ip);
```